packages/tools/kadena-cli/src/account/commands/accountAddManual.ts

```diff
@@ -0,0 +1,115 @@
+import { IS_DEVELOPMENT } from '../../constants/config.js';
+import { ensureNetworksConfiguration } from '../../networks/utils/networkHelpers.js';
+import { assertCommandError } from '../../utils/command.util.js';
+import { createCommandFlexible } from '../../utils/createCommandFlexible.js';
+import { globalOptions } from '../../utils/globalOptions.js';
+import type { Predicate } from '../types.js';
+import { addAccount } from '../utils/addAccount.js';
+import { displayAddAccountSuccess } from '../utils/addHelpers.js';
+import { getAccountDetailsForAddAccount } from '../utils/getAccountDetails.js';
+import { validateAndRetrieveAccountDetails } from '../utils/validateAndRetrieveAccountDetails.js';
+
+export const createAddAccountManualCommand = createCommandFlexible(
+  'add-manual',
+  'Add an existing account to the CLI',
+  [
+    globalOptions.accountAlias(),
+    globalOptions.accountName(),
+    globalOptions.fungible(),
+    globalOptions.networkSelect(),
+    globalOptions.chainId(),
+    globalOptions.accountOverwrite(),
+    globalOptions.publicKeys({ isOptional: false }),
+    globalOptions.predicate(),
+  ],
+
+  async (option, values) => {
+    const accountAlias = (await option.accountAlias()).accountAlias;
+    const accountName = (await option.accountName()).accountName;
+    const fungible = (await option.fungible()).fungible;
+
+    await ensureNetworksConfiguration();
+
+    const { network, networkConfig } = await option.network();
+
+    const chainId = (await option.chainId()).chainId;
+
+    const accountDetailsFromChain = accountName
+      ? await getAccountDetailsForAddAccount({
+          accountName,
+          networkId: networkConfig.networkId,
+          chainId,
+          networkHost: networkConfig.networkHost,
+        })
+      : undefined;
+
+    const isEmptyAccountDetails =
```

Comment:
This name is misleading, doesn't it mean there is an account? "Is empty" makes me think if it's true there is no account